src/libs/SidebarUtils.js

```diff
@@ -185,11 +185,29 @@ function getOrderedReportIDs(currentReportId, allReportsDict, betas, policies, p
     outstandingIOUReports.sort((a, b) => b.iouReportAmount - a.iouReportAmount || a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase()));
     draftReports.sort((a, b) => a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase()));
     if (isInDefaultMode) {
-        nonArchivedReports.sort(
-            (a, b) => new Date(b.lastVisibleActionCreated) - new Date(a.lastVisibleActionCreated) || a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase()),
-        );
+        nonArchivedReports.sort((a, b) => {
+            const stringA = a.lastVisibleActionCreated;
+            const stringB = b.lastVisibleActionCreated;
+            if (stringA < stringB) {
+                return 1;
+            }
+            if (stringA > stringB) {
+                return -1;
+            }
+            return a.displayName.toLowerCase().localeCompare(b.displayName.toLowerCase());
+        });
         // For archived reports ensure that most recent reports are at the top by reversing the order
-        archivedReports.sort((a, b) => new Date(a.lastVisibleActionCreated) - new Date(b.lastVisibleActionCreated));
+        archivedReports.sort((a, b) => {
+            const stringA = a.lastVisibleActionCreated;
+            const stringB = b.lastVisibleActionCreated;
+            if (stringA < stringB) {
+                return 1;
+            }
+            if (stringA > stringB) {
+                return -1;
+            }
+            return 0;
+        });
```

Comment:
I agree with the fix, but I think that using a variable stringA and stringB here doesn't really add anything to the code. Can we just use the properties from the object instead? 

Also, we're using the same function in both reports, so let's turn that function into a variable and reuse it in `sort(function)`

Reply:
Changed :)